### PR TITLE
feat(websessions): add support for the new web session api

### PR DESCRIPTION
**Pull request by vladikoff**
Connects to https://github.com/mozilla/fxa-content-server/issues/4585

### CODE REVIEW

**Comment by vladikoff** [comment]
@vbudhram this part ready for review

Part of this stack:

image

File: tests/ci/travis-auth-server-test.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 # Install and start the auth server
 git clone https://github.com/mozilla/fxa-auth-server.git
+git checkout web-sessions
```
Comment (vladikoff) [inline tests/ci/travis-auth-server-test.sh]: TODO: remove this before merge

File: tests/lib/session.js [hunk 2]
```diff
@@ -72,6 +72,41 @@ define([
...
+        return accountHelper.newVerifiedAccount()
+          .then(function (account) {
+            return respond(client.sessions(account.signIn.sessionToken), RequestMocks.sessions);
+          })
+          .then(
+            function (res) {
+              assert.equal(res.length, 2);
+              assert.ok(res[0].id);
+              assert.ok(res[0].userAgent);
+              assert.ok(res[0].deviceName);
```
Comment (vbudhram) [inline tests/lib/session.js]: Noticed some other values that might be useful to verify to assert in the response, `isDevice=false`, `deviceType=desktop`.

screen shot 2017-02-15 at 5 37 33 pm

**Comment by vladikoff** [comment]
>  I was wondering why there would there be two desktop devices in response since I was only using one client?

newVerifiedAccount helper does both signUp and signIn and those 2 requests do not have any state, so it is almost like 2 different clients sending those requests

**Pull request merged**